src/04-modality-specific-files/03-electroencephalography.md

```diff
@@ -23,8 +23,8 @@ Template:
 sub-<label>/
     [ses-<label>]/
       eeg/
-        sub-<label>[_ses-<label>]_task-<label>[_run-<index>]_eeg.<manufacturer_specific_extension>
-        sub-<label>[_ses-<label>]_task-<label>[_run-<index>]_eeg.json
+        sub-<label>[_ses-<label>]_task-<label>[_acq-<acquisition>][_run-<index>]_eeg.<manufacturer_specific_extension>
```

Comment:
`<acquisition>` should be `<label>`

Reply:
Thanks for the comments. I fixed it to `<label>` in the latest commit